### PR TITLE
Migrated Spring Boot demo to 1.11.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: springboot/1.11/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+* Build and push the artifact with `mvn clean package dcos:uploadArtifact`
+![Sample deploy](images/sample_deploy.png)
+
+* [Optional] Check in the Nexus UI that it was successfully uploaded.
+![Sample deploy](images/nexus_2.png)
+
+* Deploy the application using `mvn clean package dcos:uploadArtifact dcos:deployUCR`.
+Note, that the deployment will use the [template app-definition](https://github.com/dcos-labs/dcos-maven-plugin/blob/master/samples/spring-boot-sample/application.json) and replace `<NEXUS_URL>` and `<FILENAME>`
+with the respective values. You can check from the DC/OS UI that it is deploying.
```
[the remark this answers is not in the document]
Comment (pleia2) [reply springboot/1.11/README.md]: Actually, even after updating the application.json it's failing.

[ERROR] Failed to execute goal dcos:dcos-maven-plugin:0.5:deployUCR (default-cli) on project mvn-plugin-spring-sample: Execution default-cli of goal dcos:dcos-maven-plugin:0.5:deployUCR failed: org.apache.http.client.ClientProtocolException: Target host is not specified -> [Help 1]

[the remark this answers is not in the document]
Comment (pleia2) [reply springboot/1.11/README.md]: My updated application.json has the following:      "uri": "http://34.210.61.132:5581/repository/dcos/mvn-plugin-spring-sample-0.0.1-SNAPSHOT.jar",

(This was my public Nexus URL+port, and the filename as I understand it) Is that correct?

File: springboot/1.11/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+![Sample deploy](images/dcos.png)
```
[the remark this answers is not in the document]
Comment (pleia2) [reply springboot/1.11/README.md]: Looks like this should be sample_dcos.png

**Review by pleia2 — COMMENTED** [review]
I got stuck when deploying this, comments inline.

**Comment by yankcrime** [comment]
Thanks @joerg84 but I'm going to close this as it's now somewhat out of date.  If you fancy refreshing it for DC/OS 2.0 feel free to issue a new PR.

**Pull request closed without merging**